polar-wasm-api/src/polar.rs

```diff
@@ -24,6 +24,12 @@ impl Polar {
             .map_err(Error::into)
     }
 
+    #[wasm_bindgen(js_class = Polar, js_name = clearRules)]
+    pub fn wasm_clear_rules(&self) -> JsResult<()> {
+        self.0.clear_rules();
+        Ok(())
+    }
```

Comment:
```suggestion
    pub fn wasm_clear_rules(&self) {
        self.0.clear_rules()
    }
```

Reply:
What will happen now if `clear_rules()` panics?

Reply:
It'll blow up with a pretty stack trace courtesy of https://github.com/rustwasm/console_error_panic_hook